terraform/modules/monitoring/variables.tf

```diff
@@ -0,0 +1,81 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+variable "project_id" {
+  description = "The GCP project that host the pmap service."
+  type        = string
+}
+
+variable "prober_bucket_id" {
+  description = "The bucket id where prober will upload files to."
+  type        = string
+}
+
+variable "prober_bigquery_dataset_id" {
+  description = "The ID of the bigquery dataset where prober run queries from."
+  type        = string
+}
+
+variable "prober_scheduler" {
+  type        = string
+  description = "How often the prober service should be triggered, default is every 30 minutes. Learn more at: https://cloud.google.com/scheduler/docs/configuring/cron-job-schedules?&_ga=2.26495481.-578386315.1680561063#defining_the_job_schedule."
+  default     = "*/30 * * * *"
+}
+
+variable "pmap_prober_image" {
+  type        = string
+  description = "Docker image for pmap prober."
+}
+
+variable "prober_alert_threshold" {
+  type        = number
+  description = "Send alert for Prober-Service when the number of succeeded prober runs below the threshold."
+  default     = 2
+}
+
+variable "prober_alert_align_window_size_in_seconds" {
+  type        = string
+  description = "The sliding window size for counting failed prober job runs. Format example: 600s."
+  default     = "3600s"
+}
+
+variable "alert_enabled" {
+  type        = bool
+  description = "True if alerts are enabled, otherwise false."
+  default     = false
+}
+
+variable "notification_channel_email" {
+  type        = string
+  description = "The Email address where alert notifications send to."
+}
+
+variable "container_env_vars" {
```

Comment:
I don't think it is user friendly to ask users to read all the description and provide all the env vars. A better approach will be you keep your previous variable definition in the variables.tf but in the main.tf, you manually merge them into an map and them use dynamic to make the code clean.

Reply:
Updated.

Reply:
Do we even have a need for additional env vars?

Reply:
we can put the LOG_LEVEL=DEBUG in there

Reply:
Removed additional env var, and made LOG_LEVEL a variable.